src/components/SingleReview.jsx

```diff
@@ -0,0 +1,34 @@
+import { useEffect, useState } from "react"
+import { useParams } from "react-router"
+import { getReview } from "../api"
+
+export const SingleReview = () => {
+    const [review, setReview] = useState({});
+    const [isLoading, setIsLoading] = useState(true);
+    const {review_id} = useParams();
+
+    useEffect(()=>{
```

Comment:
If you setIsLoading(true) here at the start of the useEffect, if you refresh it should still show the loading again!